src/test/simple.ts

```diff
@@ -1611,6 +1611,207 @@ in
             compare(expected, actual);
             compare(expected2, actual2);
         });
+
+        it(`Record item should honor prepending Line Feed `, async () => {
+            const target: string = `[Version = "1.0.0"]
+section MultipleAuthKindConnector;
+
+[DataSource.Kind = "MultipleAuthKindConnector"]
+shared MultipleAuthKindConnector.Connects = ()=> Extension.CurrentCredential()[AuthenticationKind];
+
+MultipleAuthKindConnector = [
+    Authentication = [
+        Key = [],
+        UsernamePassword = [],
+        Windows = [],
+        Anonymous = []        
+    ]
+];`;
+
+            const expected: string = `
+[
+    Version = "1.0.0"
+]
+section MultipleAuthKindConnector;
+
+[
+    DataSource.Kind = "MultipleAuthKindConnector"
+]
+shared MultipleAuthKindConnector.Connects = () =>
+    Extension.CurrentCredential()[
+        AuthenticationKind
+    ];
+
+MultipleAuthKindConnector = [
+    Authentication = [
+        Key = [],
+        UsernamePassword = [],
+        Windows = [],
+        Anonymous = []
+    ]
+];
+`;
+
+            const expected2: string = `
+[Version = "1.0.0"]
+section MultipleAuthKindConnector;
+
+[DataSource.Kind = "MultipleAuthKindConnector"]
+shared MultipleAuthKindConnector.Connects = () => Extension.CurrentCredential()[AuthenticationKind];
+
+MultipleAuthKindConnector = [
+    Authentication = [
+        Key = [],
+        UsernamePassword = [],
+        Windows = [],
+        Anonymous = []
+    ]
+];`;
+
+            const actual: string = await expectFormat(target);
+            const actual2: string = await expectFormat(target, DefaultFormatSettingsWithMaxWidth);
+
+            compare(expected, actual);
+            compare(expected2, actual2);
+        });
+    });
+
+    it(`Record should honor its content length v1`, async () => {
+        const target: string = `(optional options as record) as table =>
+    let
+      Concat = (list) => List.Accumulate(list, null, (x, y) => if x = null then y else [Kind = "Binary", Operator = "Concatenate", Left = x, Right = y])
+    in
+      Concat`;
+
+        const expected: string = `
+(
+    optional options as record
+) as table =>
+    let
+        Concat = (
+            list
+        ) =>
+            List.Accumulate(
+                list,
+                null,
+                (
```

Comment:
This is looking a bit more like we once had with the amount of vertical whitespace. Is part of what you wanted changed @mattmasson ? Otherwise I feel like lambdas shouldn't be taking all of these lines up.

Reply:
I did not touch the lambdas within this pr. How about we enhance lambdas expression in the other pr? 😋

Reply:
Sure thing. It's another enhancement to be considered, but it's separate from this PR.